pom.xml

```diff
@@ -18,6 +18,43 @@
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
   </scm>
 
+  <properties>
+    <jenkins.version>1.596.1</jenkins.version>
+    <java.level>6</java.level>
+  </properties>
+  
+  <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-project</artifactId>
+      <version>1.6</version>
```

Comment:
:bug: AFAICT this is used only in tests, so you must specify

```xml
<scope>test</scope>
```

to avoid introducing an unwanted plugin dependency.